x/sysproxy/sysproxy_other.go

```diff
@@ -0,0 +1,13 @@
+//go:build !linux && !windows && !darwin
+
+package sysproxy
+
+// SetProxy does nothing on unsupported platforms.
+func SetProxy(ip string, port string) error {
+	return nil
```

Comment:
Maybe returning an error would be better?

```suggestion
	return errors.New("platform not supported")
```